sklearn/ensemble/_forest.py

```diff
@@ -615,6 +615,13 @@ def feature_importances_(self):
     )
     @property
     def n_features_(self):
+        """Number of features when fitting the estimator.
```

Comment:
I open https://github.com/scikit-learn/scikit-learn/pull/20414 to check the issue with sphinx and as well to discuss property formatting. We can wait to see how do we want to format property.

Reply:
I think there is no problem with docstring.